cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -88,6 +88,14 @@ message CassandraConfigRole {
     optional TaskConfig cassandraYamlConfig = 11;
     // mesos role to be used to receive resource offers
     optional string mesosRole = 12;
+
+    /**
+     * A pre-defined data directory specifying where cassandra should write it's data.
+     * NOTE:
+     * This field will be removed once MESOS-1554 is released and the framework will
+     * be able to allocate the data volume itself.
+     */
+    optional string preDefinedDataDirectory = 999999;
```

Comment:
I'd also be fine with `13`. We will run into similar issues with fields we are not yet aware of. IMO we only need a "marker" comment that says: "Hey, start with 14".